src/content/posts/2022/bootstrap-5-2-0.md

```diff
@@ -0,0 +1,159 @@
+
+- Expanded [`font-weight` utilities](https://getbootstrap.com/docs/5.2/utilities/text/#font-weight-and-italics) to include `.fw-semibold` for semibold fonts.
+
+- Expanded [`border-radius` utilities](https://getbootstrap.com/docs/5.2/utilities/borders/#sizes) to include two new sizes, `.rounded-4` and `.rounded-5`, for more options.
+
+Expect more improvements here as v5's development continues.
+
+## Responsive offcanvas
+
+Our Offcanvas component now has [responsive variations](https://getbootstrap.com/docs/5.2/components/offcanvas/#responsive). The original `.offcanvas` class remains unchanged—it hides content across all viewports. To make it responsive, change that `.offcanvas` class to any `.offcanvas-{sm|md|lg|xl|xxl}` class.
+
+## New Examples repo and guides
+
+Since the beta, we've completely rewritten our Webpack and Parcel guides—and we've also added a new Vite guide.
+
+![Guides images](/assets/img/2022/07/bootstrap-guides@2x.png)
+
+Plus, we've turned every one of those guides into a fully functioning example in our new [twbs/examples](https://github.com/twbs/examples) repo. We've even added a couple more examples to the repo, with plans to create even more.
+
+- [Starter](https://github.com/twbs/examples/tree/main/starter/) – CDN links for our CSS and JS
+- [Sass & JS](https://github.com/twbs/examples/tree/main/sass-js/) — Import Sass, Autoprefixer, Stylelint, and our JS bundle via npm
+- [Sass & ESM JS](https://github.com/twbs/examples/tree/main/sass-js-esm/) — Import Sass, Autoprefixer, and Stylelint via npm, and then load our ESM JS with a shim
+- [Webpack](https://github.com/twbs/examples/tree/main/webpack/) - Import and bundle Sass and JS with Webpack
+- [Parcel](https://github.com/twbs/examples/tree/main/parcel/) - Sass, JS via Parcel
+- [Vite](https://github.com/twbs/examples/tree/main/vite/) - Sass, JS via Vite
+
+Each guide matches up to a new example in that repo, and nearly all of them can be immeditely available in Stackblitz. Now you don't even need to have a development environment configured on your computer to get started with Bootstrap.
+
+![Examples Stackblitz repo](/assets/img/2022/07/guides-stackblitz.png)
+
+And did we mention that nearly all our code snippets now have an open in Stackblitz button?
+
+![Code snippets Stackblitz](/assets/img/2022/07/snippets-stackblitz.png)
+
+## Get the release
+
+**Head to <https://getbootstrap.com> for the latest.** It's also been pushed to npm:
+
+```sh
+npm i bootstrap@v5.2.0-beta1
+```
+
+[Read the GitHub v5.2.0-beta1 changelog](https://github.com/twbs/bootstrap/releases/tag/v5.2.0-beta1) for a complete list of changes in this release.
+
+## Support the team
+
+Visit our [Open Collective page]({{< param "opencollective" >}}) or our [team members](https://github.com/orgs/twbs/people)' GitHub profiles to help support the maintainers contributing to Bootstrap.
```

Comment:
Maybe we need to say something about:

* https://github.com/twbs/bootstrap/pull/36528
* [ experimental reserved data attribute data-bs-config](https://getbootstrap.com/docs/5.2/components/tooltips/#options)